### ISSUE TITLE
javadocs won't build with java 8

### ISSUE THREAD

**Issue by johnynek** [twitter/chill#249]
this blocks publishing chill-java

**Comment by johnynek** [comment]
```

error] /Users/oscar/oss/chill/chill-java/src/main/java/com/twitter/chill/KryoPool.java:41: warning: no @param for poolS
ize
[error]   public static KryoPool withBuffer(int poolSize,
[error]                          ^
[error] /Users/oscar/oss/chill/chill-java/src/main/java/com/twitter/chill/KryoPool.java:41: warning: no @param for ki
[error]   public static KryoPool withBuffer(int poolSize,
[error]                          ^
[error] /Users/oscar/oss/chill/chill-java/src/main/java/com/twitter/chill/KryoPool.java:41: warning: no @param for outBu
fferMin
[error]   public static KryoPool withBuffer(int poolSize,
[error]                          ^
[error] /Users/oscar/oss/chill/chill-java/src/main/java/com/twitter/chill/KryoPool.java:41: warning: no @param for outBu
fferMax
[error]   public static KryoPool withBuffer(int poolSize,
[error]                          ^
[error] /Users/oscar/oss/chill/chill-java/src/main/java/com/twitter/chill/KryoPool.java:41: warning: no @return
[error]   public static KryoPool withBuffer(int poolSize,
[error]                          ^
[error] /Users/oscar/oss/chill/chill-java/src/main/java/com/twitter/chill/KryoPool.java:55: warning: no @param for poolS
ize
[error]   public static KryoPool withByteArrayOutputStream(int poolSize,
[error]                          ^
[error] /Users/oscar/oss/chill/chill-java/src/main/java/com/twitter/chill/KryoPool.java:55: warning: no @param for ki
[error]   public static KryoPool withByteArrayOutputStream(int poolSize,
[error]                          ^
[error] /Users/oscar/oss/chill/chill-java/src/main/java/com/twitter/chill/KryoPool.java:55: warning: no @return
[error]   public static KryoPool withByteArrayOutputStream(int poolSize,
[error]                          ^
```

**Comment by ianoc** [comment]
Fixed in #250
